Classes/Database/Schema/SchemaUpdateInterface.php

```diff
@@ -0,0 +1,43 @@
+<?php
+namespace Helhum\Typo3Console\Database\Schema;
+
+/*
+ * This file is part of the TYPO3 console project.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ *
+ * For the full copyright and license information, please read
+ * LICENSE file that was distributed with this source code.
+ *
+ */
+
+/**
+ * The database schema update contract
+ */
+interface SchemaUpdateInterface
```

Comment:
I don't have good arguments beside that I try to move to a more imperative design, rather a procedural design. Creating objects as actors supports staying procedural.

I'm perfectly aware, that this does not happen by just choosing different names but still letting the objects be actors. But it helps me personally to get my mind shifted away from procedural coding